packages/react-charts/src/components/ChartContainer/ChartContainer.tsx

```diff
@@ -141,12 +142,16 @@ export interface ChartContainerProps extends VictoryContainerProps {
    * is given, the width prop from the child component passed will be used.
    */
   width?: number;
+  ouiaComponentType?: string;
```

Comment:
Please keep props in alphabetical order and add comments. This will be output in the docs.

Please add the following to the comment -- see `sharedEvents` prop.

`**This prop should not be set manually.**`